src/entries/popup/pages/home/Tokens.tsx

```diff
@@ -123,22 +133,16 @@ export function Tokens() {
     },
   );
 
-  const allAssets = useMemo(
-    () =>
-      uniqBy(
-        [...assets, ...customNetworkAssets].sort(
-          (a: ParsedUserAsset, b: ParsedUserAsset) =>
-            parseFloat(b?.native?.balance?.amount) -
-            parseFloat(a?.native?.balance?.amount),
-        ),
-        'uniqueId',
-      ),
+  const combinedAssets = useMemo(
+    () => [...assets, ...customNetworkAssets],
     [assets, customNetworkAssets],
   );
 
+  const filteredAssets = useFilteredPinnedAssets(combinedAssets);
```

Comment:
Pinned tokens should be on top and the rest of tokens should appear after pinned tokens. Just a small filter function.